src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PropertyPages/DebugPageControl.xaml

```diff
@@ -413,7 +414,8 @@
                 IsEnabled="{Binding Path=IsProfileSelected, Mode=OneWay}"
                 Visibility="{Binding Path=SupportsEnvironmentVariables, Mode=OneWay, Converter={StaticResource visibilityConverter}}"
                 ItemsSource="{Binding EnvironmentVariables, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}"
-                VerticalGridLinesBrush="{DynamicResource VsBrush.GridLine}" HorizontalGridLinesBrush="{DynamicResource VsBrush.GridLine}" 
+                VerticalGridLinesBrush="{DynamicResource {x:Static shell:VsBrushes.GridLineKey}}"
+                HorizontalGridLinesBrush="{DynamicResource {x:Static shell:VsBrushes.GridLineKey}}"
```

Comment:
These failed to resolve before and after this change, giving black grid lines:

![image](https://user-images.githubusercontent.com/350947/93829658-76c11e00-fcb1-11ea-8d39-98e4996f523f.png)

Filed https://dev.azure.com/devdiv/DevDiv/_workitems/edit/1217712